src/cheatsheet_vm.md

```diff
@@ -0,0 +1,104 @@
+# Quick deploy on Ubuntu 20.04 VM
+
+> /!\ This configuration is totally not suitable for production environment!
+
+```sh
+# Check the machine
+uname -a
```

Comment:
Not a bad idea to start with this, but the question is: "so what?" 😄 
Maybe add a cmoment that if the system replies "Windows" of some sort then the user should be worried with the rest... :)

Reply:
Thank you @bowni for the feedbacks! I have updated the document accordingly!